sections/webappapis.include

```diff
@@ -159,33 +181,116 @@
 
 <h5 id="sec-fetching-scripts"><dfn>Fetching scripts</dfn></h5>
 
-  The various script-fetching algorithms below have two hooks that may be customized by their
-  callers:
+  <p>This section introduces a number of algorithms for fetching scripts, taking various necessary
+  inputs and resulting in <a>classic scripts</a> or <a>module scripts</a>.</p>
+
+  <hr />
+
+  <p><dfn>Script fetch options</dfn> is a <a>struct</a> with the following <a>items</a>:</p>
+
+  <dl>
+   <dt><dfn for="script">cryptographic nonce</dfn></dt>
+
+   <dd><p>The <a>cryptographic nonce metadata</a> used
+   for the initial fetch and for fetching any imported modules</p></dd>
+
+   <dt><dfn for="script">integrity metadata</dfn></dt>
+
+   <dd><p>The <a>integrity metadata</a> used for
+   the initial fetch</dd>
+
+   <dt><dfn for="script">parser metadata</dfn></dt>
 
-  * <dfn>Set up the request</dfn>, which takes a <a>request</a> which it may modify before the
-     algorithm continues
-  * <dfn>Process the response</dfn>, which takes a <a>response</a> and must either return
-     <code>true</code> or <code>false</code> to indicate success or failure, respectively
+   <dd><p>The <a>parser metadata</a> used for the
+   initial fetch and for fetching any imported modules</p></dd>
+
+   <dt><dfn for="script">credentials mode</dfn></dt>
+
+   <dd><p>The <a>credentials mode</a> used for the
+   initial fetch (for <a>module scripts</a>) and for fetching any
+   imported modules (for both <a>module scripts</a> and <a>classic scripts</a>)</p></dd>
+
+   <dt><dfnfor="script">referrer policy</dfn></dt>
+
+   <dd><p>The <a>referrer policy</a> used for the
+   initial fetch and for fetching any imported modules</p></dd>
+  </dl>
+
+  <p class="note">Recall that via the <code>import()</code> feature, <a>classic scripts</a> can import <a>module scripts</a>.</p>
+
+  
+  <p>The <dfn>default classic script fetch options</dfn> are a set of<a>script fetch options</a>
+  whose <a>cryptographic nonce</a> is the empty
+  string, <a>integrity metadata</a> is the
+  empty string, <a>parser metadata</span> is "<code>not-parser-inserted</code>", <a>credentials mode</a> is "<code>omit</code>", and <a>referrer
```

Comment:
please split the line differently so you don't have a line break inside the link `<a>referrer policy</a>`